ci/scripts/run-test-network.sh

```diff
@@ -0,0 +1,37 @@
+set -euo pipefail
+
+function executeApplication() {
+  pushd "${1}"
+  node invoke.js && node query.js
+  popd
+}
+
+./network.sh up createChannel -ca -s couchdb -i ${FABRIC_VERSION} # FABRIC_VERSION is set in ci/azure-pipelines.yml
+
+# Setup javascript application
+pushd ../asset-transfer-basic/application-javascript
+npm install
+node enrollAdmin.js && node registerUser.js
+popd
+
+# Setup typescript application
+pushd ../asset-transfer-basic/application-typescript
+npm install -g typescript
+npm install
+cd src
+tsc enrollAdmin.ts invoke.ts query.ts registerUser.ts
+node enrollAdmin.js && node registerUser.js
```

Comment:
Not going to run this as the user has already been registered above.
Options are:
- do not run it and wait till I make enroll and register common
- stop and start the network before each application
- have each application create their own user with unique name

Reply:
I moved to just copying the wallet for now. Since enroll and register are SDK functionality no benefit to testing them twice